deployment-templates/compute-engine/client/forseti-instance-server.py

```diff
@@ -0,0 +1,257 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
This file is for server and should not be here in the `client` directory.

Reply:
Hmmm not sure how this happened. Might've occurred when I got behind the dev branch. I'll fix it.